source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +235,63 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', async () => {
+      if (process.platform === 'darwin') {
+        if (this._tray == null) {
+          let basepath = app.getAppPath()
+          let image = await nativeImage.createThumbnailFromPath(basepath + '/source/main/assets/icons/black_white_128.png', { width: 16, height: 16 })
+          this._tray = new Tray(image)
```

Comment:
1. Why scale the image from 128x128 to 16x16? May I suggest create the image as 16x16, then no scaling is necessary. Then it is also possible to use less code by

   ```typescript
   this._tray = Tray(path.join(__dirname, 'assets/icons/???.png'))
   ```

1. The [tray design](https://github.com/UofA-SPI21-team7/zettlr-assignment/wiki/Tray-design) on Mac uses a white logo with transparent background. The `black_white_128x128.png` is a black background.

1.  `/source/` will not work when Zettlr is [packaged](https://github.com/UofA-SPI21-team7/zettlr-assignment/wiki/Zettlr-creating-packages). See #18 on how the Windows Icon works for both development and packages.